cpp/src/strings/regex/regcomp.cpp

```diff
@@ -570,182 +570,180 @@ class regex_parser {
  * @brief The compiler converts class list into instructions.
  */
 class regex_compiler {
-  reprog& m_prog;
-
-  struct Node {
+  struct and_node {
```

Comment:
Is the implication of this rename that nodes represent conditions in the regex that must all be true (i.e. a logical and)?